api/app/routers/pteams.py

```diff
@@ -1398,14 +1398,33 @@ def get_pteam_members(
         raise NO_SUCH_PTEAM
     if not check_pteam_membership(pteam, current_user):
         raise NOT_A_PTEAM_MEMBER
-    return pteam.members
+
+    pteam_members = []
+    for member in pteam.members:
+        for pteam_role in member.pteam_roles:
+            if pteam_role.pteam_id == str(pteam_id):
+                is_admin = pteam_role.is_admin
+                break
```

Comment:
ないほう表記を使ってほうがわかりやすい気もします
```suggestion
        is_admin = any(role.pteam_id == str(pteam_id) and role.is_admin for role in member.pteam_roles)
```

Reply:
any() は単に一致するroleにis_admin=Trueが1つでもあるかを判定するだけなので、next()を使う方が正確かと思いますし、内包表記の方が分かりやすいと思います。

Reply:
全て内包表記にするのは分かりづらいかと思い、is_adminの部分のみ内包表記にしました